Sources/StreamChatSwiftUI/ChatChannel/MessageList/MessageListConfig.swift

```diff
@@ -54,6 +56,7 @@ public struct MessageListConfig {
     public let maxTimeIntervalBetweenMessagesInGroup: TimeInterval
     public let cacheSizeOnChatDismiss: Int
     public let iPadSplitViewEnabled: Bool
+    public let embedInNavigationView: Bool
```

Comment:
The `MessageListConfig` is mostly about configs related to the message list. We should move the `embedInNavigationView` as a parameter in the initializer of the `ChatChannelListView`, with the default value of true.

Reply:
Moved this to the initializer of `ChatChannelListView` (as requested)!